>>> 'Pkg.add("LevelDB")' log
INFO: Installing LevelDB v0.0.1
INFO: Package database updated

>>> 'using LevelDB' log
Julia Version 0.4.0-dev+765
Commit 5bc820e (2014-09-25 01:04 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3

>>> test log
ERROR: error compiling open_db: could not load module libleveldb: libleveldb: cannot open shared object file: No such file or directory
 in include at ./boot.jl:246
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
 in _start_3B_3594 at /home/idunning/julia04/usr/bin/../lib/julia/sys.so
while loading /home/idunning/pkgtest/.julia/v0.4/LevelDB/test/runtests.jl, in expression starting on line 4

INFO: Testing LevelDB
===============================[ ERROR: LevelDB ]===============================

failed process: Process(`/home/idunning/julia04/usr/bin/julia /home/idunning/pkgtest/.julia/v0.4/LevelDB/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
INFO: No packages to install, update or remove
ERROR: LevelDB had test errors
 in error at error.jl:21
 in test at pkg/entry.jl:719
 in anonymous at pkg/dir.jl:28
 in cd at ./file.jl:20
 in cd at pkg/dir.jl:28
 in test at pkg.jl:68
 in process_options at ./client.jl:213
 in _start at ./client.jl:354
 in _start_3B_3594 at /home/idunning/julia04/usr/bin/../lib/julia/sys.so


>>> end of log
